base/src/test/java/com/fasterxml/jackson/jakarta/rs/base/cfg/AnnotationBundleKeyTest.java

```diff
@@ -1,6 +1,8 @@
 package com.fasterxml.jackson.jakarta.rs.base.cfg;
 
 import java.lang.annotation.Annotation;
+import java.util.Arrays;
+import java.util.Comparator;
```

Comment:
Can you check if there are ways to sort Annotations without importing Comparator or Arrays?

Reply:
I believe we can import streams and sort the annotations but I don't understand the usecase not to import Comparator or arrays since it is available in java.util package itself and we don't need any third-party packages

Reply:
I was suggesting this so that this decreases changes by number of lines of codes, I am not sure if it is possible to do so